MessageBroker.js

```diff
@@ -392,6 +392,7 @@ class MessageBroker {
 			if(msg.eventType=="custom/myVTT/delete_token"){
 				let tokenid=msg.data.id;
 				if(tokenid in window.TOKEN_OBJECTS)
+					window.TOKEN_OBJECTS[tokenid].options.deleteableByPlayers = true;
```

Comment:
The early return at the beginning of Token.delete was introduced when the `deletableByPlayers` feature was introduced. I think we should just roll back that bit of logic instead of working around it here. We'll need to double check everywhere that calls `delete` to make sure we don't accidentally allow players to delete them. If that gets too messy, and this this is actually the cleanest option, then what you have here is good.